.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+- package-ecosystem: "pip"
+  directory: "/"
+  schedule:
+    interval: "daily"
```

Comment:
If it's all the same to you, I'd rather this schedule was more gentile. Porch doesn't have a CD process, so there would be far more churn than releases and deployments. I'd suggest weekly or monthly would be quite enough to ward off evil.

Reply:
Then I'd suggest acting on them weekly or monthly, but leaving the daily schedule so that if a "critical" pops up it can dealt with. The actual frequency of PRs is much less than daily (typically 2-3 per month), so best case is you see the 3 PRs at the end of the month, worst case is you sit on a critical CVE for that same time.

Reply:
Very well. Thanks for enabling it in any event. Been collecting dust in my to-do for years.